### PR TITLE
xapi/tipb: remove points in select request.

### PR DESCRIPTION
**Pull request by coocood**
Points was introduced to reduce the data size if there are many point ranges.
But it makes code more complex than expected, and the reduced size is trivial compares to the total package size.

Remove it to simplify code.

### CODE REVIEW

File: executor/executor_xapi.go [hunk 1]
```diff
@@ -200,7 +200,13 @@ func (e *XSelectIndexExec) doTableRequest(txn kv.Transaction, handles []int64) (
 	selTableReq.TableInfo = tablecodec.TableToProto(e.indexPlan.Table)
 	selTableReq.Fields = resultFieldsToPBExpression(e.indexPlan.Fields())
 	for _, h := range handles {
-		selTableReq.Points = append(selTableReq.Points, codec.EncodeInt(nil, h))
+		pbRange := new(tipb.KeyRange)
+		pbRange.Low = codec.EncodeInt(nil, h)
+		if h != math.MaxInt64 {
+			h++
```
Comment (shenli) [inline executor/executor_xapi.go]: Do not need to append to ranges.

File: executor/executor_xapi.go [hunk 2]
```diff
@@ -214,55 +220,41 @@ func resultFieldsToPBExpression(fields []*ast.ResultField) []*tipb.Expr {
...
-				hiVal++
-			}
-			hr.High = codec.EncodeInt(nil, ran.HighVal)
-			hrs = append(hrs, hr)
+	for _, tableRange := range tableRanges {
+		pbRange := new(tipb.KeyRange)
+		pbRange.Low = codec.EncodeInt(nil, tableRange.LowVal)
+		hi := tableRange.HighVal
+		if hi != math.MaxInt64 {
+			hi++
```
Comment (shenli) [inline executor/executor_xapi.go]: Do not need to append to ranges.

Comment (coocood) [inline executor/executor_xapi.go]: This is high value, we need to append the range.

**Comment by shenli** [comment]
LGTM

**Comment by zimulala** [comment]
LGTM

**Pull request merged**